src/chains/models.py

```diff
@@ -24,6 +26,12 @@ def native_currency_path(instance: "Chain", filename: str) -> str:
     return f"chains/{instance.id}/currency_logo{file_extension}"
 
 
+def validate_native_currency_size(image: str | IO[bytes]) -> None:
+    image_width, image_height = get_image_dimensions(image)
+    if image_width > 512 or image_height > 512:
```

Comment:
Should we restrict the ratio too? meaning `and image_width == image_height` ? Seems to me that this is expected by the clients, but I don't know it for a fact. Pointing it out just in case :)

Reply:
Yeah that's a good point. I will review the images and check if we have any with a different ratio